android/datastore.md

```diff
@@ -66,238 +65,245 @@ type Post @model {
-            Log.e("Error: ", Log.getStackTraceString(error));
-        }
+    @Override
+    public void onError(Throwable error) {
+        Log.e("DataStore", "Error.", error);
     }
-);
+});
 ```
 
 # Delete Data
 
-To delete an item simply pass in an instance:
+To delete an item, simply pass in an instance:
 
 ```java
 ResultListener<DataStoreItemChange<Post>> deleteListener = new ResultListener<DataStoreItemChange<Post>>() {
     @Override
     public void onResult(DataStoreItemChange<Post> result) {
-        Log.i("Result: ", result.item());
+        Log.i("DataStore", "Result: " + result.item());
     }
 
     @Override
     public void onError(Throwable error) {
-        Log.e("Error: ", Log.getStackTraceString(error));
+        Log.e("DataStore", "Error.", error);
     }
 };
 
 Amplify.DataStore.query(Post.class, Post.ID.eq("123"), new ResultListener<Iterator<Post>>() {
-        @Override
-        public void onResult(Iterator<Post> result) {
-            if (result.hasNext()) {
-                Post post = result.next();
-                Amplify.DataStore.delete(post, deleteListener);
-            }
+    @Override
+    public void onResult(Iterator<Post> result) {
+        if (result.hasNext()) {
+            Post post = result.next();
+            Amplify.DataStore.delete(post, deleteListener);
         }
+    }
 
-        @Override
-        public void onError(Throwable error) {
-            Log.e("Error: ", Log.getStackTraceString(error));
-        }
+    @Override
+    public void onError(Throwable error) {
+        Log.e("DataStore", "Error.", error);
     }
-);
+});
 ```
 
 # Observe Data
 
-`AWSDataStorePlugin.observe()` method takes dependency on RxJava and returns an instance of `io.reactivex.Observable<DataStoreItemChange<? extends Model>>` when a model class is observed. You will need to add `implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'` to your `build.gradle`.
-{: .callout .callout--info}
+You can subscribe to changes on your Models by using the DataStore's `observe` method. This method reacts dynamically to all changes in local storage. These changes could be the result of local modifications, or the result of changes observed on your GraphQL endpoint, if using remote synchronization.
 
-You can subscribe to changes on your Models by using `observe` in the DataStore API. This reacts dynamically to updates of data to the underlying Storage Engine, which could be the result of GraphQL Subscriptions as well as Queries or Mutations that run against the backing AppSync API if you are synchronizing with the cloud.
+The `AWSDataStorePlugin.observe()` method returns an `io.reactivex.Observable<DataStoreItemChange<? extends Model>>`. You can subscribe to this Rx Observable to get notifications whenever any model is created, updated, or deleted. To use a method in this family, you must make Rx available at runtime by adding `implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'` to your module's `build.gradle`.
```

Comment:
Should we hyperlink to the Rx website?

Reply:
Sure, sounds like a good addition -- will add.